QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150511_002010	$I shall name you Revelator, my representative sent out to seek help from others. {nl}We can't be deceived any more. {nl}We have to protest against the violence behind the mask that claims to protect us.
+QUEST_20150511_002011	$We believe your help is the main factor to our success so that our efforts were not made in vain.
+QUEST_20150511_002012	$We've finally arrived in front of the Zachariel's Mausoleum. {nl}He was the first king of this kingdom and used an enormous number of people to make it. {nl}I believe only this place can answer the question I've had in my mind for a long time.
+QUEST_20150511_002013	$What was he trying to protect with this gigantic tomb? {nl}The answer must be inside, but the bleak atmosphere the inside exudes makes me hesitate to go in.
+QUEST_20150511_002014	$There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pole outside of the tomb.
+QUEST_20150511_002015	$With the help from the young, kind historian, I was able to find a big clue. {nl}Yes. The best place to hide a tree is in a forest. {nl}Ruko Plateau and the Plateau of Rex… I'm so close I can almost touch it. {nl}I'm confident the Goddess sent him to me. May the blessing of the Goddess always be with us!
+QUEST_20150511_002016	$Some poles unlock the tomb and some altars lock it. {nl}The magical energy that surrounds the tomb moves by the key. {nl}But none can be that key. {nl}We are not chosen.
+QUEST_20150511_002017	$The curse disturbs the rules of the universe that are being controlled by the Goddess with human greed.{nl}These rules in the human world get influenced by the personalities of each human, their abilities and their potential power. {nl}The cases when these abilities get released depend on the conditions above.
+QUEST_20150511_002018	$According to the old writing that is left on the great cathedral, that is using a small part of the Goddess' abilities secretly.{nl}Some people say that after seducing weak minds from humans, the demons used them for the source of the curse. {nl}May the Goddess bless!
+QUEST_20150511_002019	$There are two ways to release the curse: {nl}Remove the curse using a greater power or to eliminate the caster of the curse. {nl}
+QUEST_20150511_002020	$Removing the curse with a greater power: {nl}This way depends on whether the power that overwhelms the curse comes from good or evil and from darkness or light.{nl}{nl}The most important factor for eliminating the caster: {nl}Even after eliminating the caster, the curse may not be removed due to other factors.{nl}{nl}I have no doubt that the power of the Goddess is stronger.
```

Comment:
should be "using a greater power" to keep uniformity with the previous sentence.

> This way depends on whether the power that overwhelms the curse comes from good or evil and from darkness or light.

Is good/light and evil/darkness different things in ToS universe? The whole enumeration part sounds awkward. Maybe checking the translation is a good idea?